.gitignore

```diff
@@ -6,3 +6,6 @@ npm-debug.log
 .tern-project
 yarn-error.log
 .vscode/
+manifest.yml
+.imdone/
+config.local.yml
```

Comment:
@tortilaman let's leave that value out since it's not a standard file, but the other two can stay.